README.md

```diff
@@ -4,10 +4,10 @@
 "As a (*type of person*), I want (*a goal*) so that (*a reason*)."
 
 ## Feature Backlog (by Priority)
-- [ ] Feature 1
-- [ ] Feature 2
-- [ ] Feature 3
-- [ ] Feature 1
+- [ get the design done ] Feature 1
```

Comment:
Very good 👍 Don't forget to "check these off" as you complete them.
The Markdown will look like this:
```
[x] get the design done
[x] second finished feature name
[ ] unfinished feature name
```